mlflow/store/tracking/abstract_store.py

```diff
@@ -271,3 +271,20 @@ def log_batch(self, run_id, metrics, params, tags):
         :return: None.
         """
         pass
+
+    @abstractmethod
+    def record_logged_model(self, run_id, mlflow_model):
+        """
+        Record logged model information with tracking store. The list of logged model infos is
+        maintained in a mlflow.models tag in json format.
```

Comment:
```suggestion
        maintained in a mlflow.models tag in JSON format.
```

Is it worth specifying the initial schema of logged model info?